docs/en/sql-reference/statements/show.md

```diff
@@ -208,7 +208,7 @@ The optional keyword `FULL` causes the output to include the collation, comment
 The statement produces a result table with the following structure:
 - field - The name of the column (String)
 - type - The column data type (String)
-- null - If the column data type is Nullable (UInt8)
+- null - `YES` if the column data type is Nullable, `NO` otherwise (UInt8)
```

Comment:
it's TEXT now I think (not UInt8)

Reply:
It's actually `String` in ClickHouse lingo. Please just change it.